.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: php
+
+dist: trusty
+
+env:
+  global:
+    - COMPOSER_ROOT_VERSION=1.x-dev
+
+matrix:
+  include:
+    - php: 5.6
+    - php: 7.0
+      env: DB=PGSQL
+    - php: 7.1
+    - php: 7.2
+
+before_script:
+  # Init PHP & increase speed by removing xdebug
+  - phpenv rehash
+  - phpenv config-rm xdebug.ini
+
+  # Install composer dependencies
+  - composer validate
+  - if [[ "$DB" == "PGSQL" ]]; then composer require --no-update silverstripe/postgresql:2.0.x-dev; fi
+  - composer install --prefer-dist --no-interaction --no-progress --no-suggest --optimize-autoloader --verbose --profile
```

Comment:
You'll need to `--prefer-source` to install the modules with their tests, since they're generally excluded via gitattributes

Reply:
hmm, seems travis is green; but I'll make the change anyway as it's the right thing to do :)

Reply:
Yeah, but I don't think it's running any tests

Reply:
Hmm, thought that might be the case. I'll look into it :)